### PR TITLE
Feature/pr ens

### PR DESCRIPTION
**Pull request by jkuang-exploro**
updated letkf_only.yaml; add FIX_SCRUB option for user.yaml

user.yaml need to be re-generated.

### CODE REVIEW

**Comment by guillaumevernieres** [comment]
That branch probably needs a merge of develop ...

**Comment by jkuang-exploro** [comment]
I've merged the develop into this branch. Now should be OK.

**Comment by jkuang-exploro** [comment]
@guillaumevernieres Please kindly re-test it andsee if it's OK :)

**Comment by guillaumevernieres** [comment]
> @guillaumevernieres Please kindly re-test it andsee if it's OK :)

Will do asap and before the end of the day, busy with other things right now.

File: workflow/platforms/hera.yaml [hunk 1]
```diff
@@ -128,18 +128,21 @@ platform: !Platform
...
-      - do: /scratch1/NCEPDEV/stmp4
-        when: !calc ( int(tools.can_write(do)) and tools.panasas_gb(do) )
-      - do: /scratch2/NCEPDEV/stmp1
-        when: !calc ( int(tools.can_write(do)) and tools.panasas_gb(do) )
-      - do: /scratch2/NCEPDEV/stmp3
-        when: !calc ( int(tools.can_write(do)) and tools.panasas_gb(do) )
-      - do: !expand "{doc.user_places.PROJECT_DIR}"
-        when: !calc tools.panasas_gb(do)/4
-        message: "{do}: use project directory as scrub space"
+    - !FirstTrue
```
Comment (flampouris) [inline workflow/platforms/hera.yaml:131]: What's the difference between the FirstMax and FirstTrue?

**Review by flampouris — APPROVED** [review]

**Comment by flampouris** [comment]
It went through 2 cycles, successfully I am merging.

File: workflow/platforms/hera.yaml (hunk 1 above)
Comment (jkuang-exploro) [reply workflow/platforms/hera.yaml:131]: It's basically what it sounds like: selecting the first true value; or selecting the max value.

**Pull request merged**